common/src/main/java/com/linecorp/centraldogma/common/PathPatternOption.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2021 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.centraldogma.common;
+
+import java.util.function.Function;
+
+/**
+ * A {@link PathPatternBuilder} option.
+ */
+final class PathPatternOption {
```

Comment:
I'm not sure if this class is needed at all. Why do we need to create a `PathPattern` lazily? Can't we just create it as early as possible?

Reply:
I have merged `PathPatternOptions` and `PathPatternOption` with `PathPatternBuilder`